test/integration/functional_test.go

```diff
@@ -494,7 +494,25 @@ func validateLogsCmd(ctx context.Context, t *testing.T, profile string) {
 
 // validateProfileCmd asserts "profile" command functionality
 func validateProfileCmd(ctx context.Context, t *testing.T, profile string) {
-	rr, err := Run(t, exec.CommandContext(ctx, Target(), "profile", "list"))
+	// Profile command should not create a nonexistent profile
+	nonexistentProfile := "lis"
+	rr, err := Run(t, exec.CommandContext(ctx, Target(), "profile", nonexistentProfile))
+	if err != nil {
+		t.Errorf("%s failed: %v", rr.Args, err)
+	}
+	for _, line := range []string{fmt.Sprintf("Created a new profile : %s", nonexistentProfile), fmt.Sprintf("minikube profile was successfully set to %s", nonexistentProfile)} {
```

Comment:
how about instead of parsing the output, do a 
"minikube profile list --output=json" and expect not to see the "lis" in the created profiles?

Reply:
Absolutely! I’ll update it now